src/simulation/dynamics/_GeneralModuleFiles/svIntegratorRK4.cpp

```diff
@@ -43,10 +43,12 @@ void svIntegratorRK4::integrate(double currentTime, double timeStep)
     int i;              // dynamic object loop counter
 
     for (i=0; i<dynPtrs.size(); i++) {
```

Comment:
I subscribe to this change. I was wondering if we should use `const &auto` instead? It would guarantee that we do not change the value of the variable being iterated and would speed up the loop by passing by reference instead of by value.

Reply:
Yes! You are correct.

Reply:
You want me to make this change now, or in #222 refactor.

Reply:
Is the `&auto` a typo here?  Should be `auto`?

Reply:
It's `auto&`

Reply:
I'm not able to make this work right now.  We can do this with #222 if we want to refine this further.